### PR TITLE
Cleanup etcd entries in parallel.

### PR DESCRIPTION
**Pull request by AlCutter**

### CODE REVIEW

File: cpp/server/ct-server.cc [hunk 1]
```diff
@@ -224,34 +225,50 @@ static const bool sign_dummy =
...
                      const MasterElection* election) {
+  CHECK_NOTNULL(tree_signer);
+  CHECK_NOTNULL(election);
   const steady_clock::duration period(
       (seconds(FLAGS_sequencing_frequency_seconds)));
   steady_clock::time_point target_run_time(steady_clock::now());
 
   while (true) {
     if (election->IsMaster()) {
-      const ScopedLatency sequence_run_latency(
```
Comment (pphaneuf) [inline cpp/server/ct-server.cc]: Why lose this latency metric?

**Comment by pphaneuf** [comment]
LGTM

File: cpp/server/ct-server.cc (hunk 1 above)
Comment (AlCutter) [inline cpp/server/ct-server.cc]: Just because it'd now be measuring the same thing as `sequencer_delete_latency_ms` give or take a couple of µs

**Comment by AlCutter** [comment]
Fixes #605

**Pull request merged**